extensions/wikia/Email/EmailController.class.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+namespace Email;
+
+abstract class EmailController extends \WikiaController {
+	const DEFAULT_TEMPLATE_ENGINE = \WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
+
+	/** @var \User The user associated with the current request */
+	protected $currentUser;
+
+	/** @var \User The user this email will be sent to */
+	protected $targetUser;
```

Comment:
Consider adding getters for `$currentUser` and `$targetUser`. Maybe I've been overdosing on Java lately :)